src/edu/ucsb/cs56/projects/music/beatbox/BeatBoxFinal.java

```diff
@@ -228,6 +299,10 @@ public void actionPerformed(ActionEvent a) {
         public void actionPerformed(ActionEvent a) {
 	    float tempoFactor = sequencer.getTempoFactor() ;
 	    sequencer.setTempoFactor((float) (tempoFactor * 1.03) ) ;
+        float temp = (float) (tempoFactor * 1.03);
+        //float tempoFactor1 = sequencer.getTempoInBPM();
+        String tempo = Float.toString(temp * sequencer.getTempoInBPM());
```

Comment:
can you leave a comment here explaining why this is correct? I thought this would be wrong since it looks like you're multiplying by the 1.03 twice, but I measured the BPM and it looks like the right value is being displayed.

Reply:
also once again remove the commented lines.